Formula/latexml.rb

```diff
@@ -12,11 +12,83 @@ class Latexml < Formula
     sha256 "b911ac9897012edcc7c32d96785e4ca3830ce8cbddff78da0942263c7fb0d0bb" => :sierra
   end
 
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
+  uses_from_macos "perl"
+  uses_from_macos "pkg-config"
```

Comment:
```suggestion
  depends_on "pkg-config"
```

pkg-config isn't shipped with macOS.

Reply:
Thanks. Do we have a list of what kinds of things `uses_from_macos` is supposed to contain? I thought it was a drop-in, blanket replacement for `depends_on “formula” unless OS.mac?`

Reply:
It's supposed to be for things that the formula build uses from the macOS system, which ships with perl, libxml, and libxslt. Usually what I do is on macOS run a `brew info` to see if it's provided by macOS. 

When I get around to finishing https://github.com/Homebrew/brew/pull/6265 there should be an audit for that.